include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -693,6 +693,60 @@ def TTIR_UnsqueezeOp : TTIR_DPSOp<"unsqueeze"> {
     let hasVerifier = 1;
 }
 
+def TTIR_ClampOp : TTIR_DPSOp<"clamp"> {
+    let summary = "Clamp op.";
+    let description = [{
+      Clamp tensor values to a specified range.
+
+      Example:
+        min: 2.000000+00
+        input: [[0, 1, 2, 3, 4, 5, 6, 7]]
+        max: 5.000000+00
+
+        "ttir.clamp"(%arg0) <{max = 4.000000e+00 : f32, min = 4.000000e+00 : f32}>
+        -> %out = [[2, 2, 2, 3, 4, 5, 5, 5]]
```

Comment:
This example doesn't seem right.

If `%arg0 = [[0, 1, 2, 3, 4, 5, 6, 7]]` and `max = 4.0` and `min = 4.0` then `%out = [[4, 4, 4, 4, 4, 4, 4, 4]]`.

If `%arg0 = [[0, 1, 2, 3, 4, 5, 6, 7]]` and `max = 5.0` and `min = 2.0` then `%out = [[2, 2, 2, 3, 4, 5, 5, 5]]`.

Reply:
Thanks a lot for pointing it. I have fixed the example.